pt.html

```diff
@@ -201,96 +207,97 @@
 <panel w=500 h=450 fadeInOn="flip_spaced_rep">
 <pic src="pics/intro0.png" sx=950 sy=0></pic>
 <words x=10 y=10 w=430 h=60>
-	And it's <i>awesome</i>.
-	I started using Spaced Repetition earlier this year to learn French.
+	E é <i>incrível</i>.
+	Comecei a usar o Repetição Espaçada no início deste ano para aprender francês.
```

Comment:
Não encaixa no balão.

Acho que seria **a** Repetição Espaçada. 

Acho que poderia tirar o `no início` para diminuir a frase.